core/App/screens/CredentialDetails.tsx

```diff
@@ -185,6 +184,12 @@ const CredentialDetails: React.FC<CredentialDetailsProps> = ({ navigation, route
                 <Text style={[TextTheme.title]}>{t('CredentialDetails.IssuedBy')}</Text>{' '}
                 <Text style={[TextTheme.normal]}>{credentialConnectionLabel}</Text>
               </Text>
+              <Text>
+                <Text style={[TextTheme.title]}>{t('CredentialDetails.Issued')}:</Text>{' '}
```

Comment:
I like the format you used in another file:

`{t('CredentialDetails.Issued') + ' '} {formatTime(credential.createdAt)}`

Could probably even shorten it to:

<pre>`{${t('CredentialDetails.Issued')} ${formatTime(credential.createdAt)}}`</pre>